optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/SingleBenchmarkResult.java

```diff
@@ -60,6 +61,7 @@
     private Score score = null;
     private long timeMillisSpent = -1L;
     private long calculateCount = -1L;
+    private EnvironmentMode environmentMode = null;
```

Comment:
The problem is, if the solver config has no environment mode set, the default doesn't get reflected in the config, therefore `solverBenchmarkResult.getSolverConfig().getEnvironmentMode()` is always null, f.e. in the GeneralOptaPlannerBenchmark.